generator/internal/language/templates/rust/crate/src/builders.rs.mustache

```diff
@@ -64,7 +65,21 @@ impl {{NameToPascal}} {
     pub async fn send(self) -> Result<{{OutputTypeName}}> {
         self.0.stub.{{NameToSnake}}(self.0.request, self.0.options).await
     }
+    {{#IsPageable}}
 
+    /// Streams the responses back.
+    #[cfg(feature = "unstable-stream")]
+    pub async fn stream(self) -> gax::paginator::Paginator<{{OutputTypeName}}, gax::error::Error> {
+        let token = gax::paginator::extract_token(&self.0.request.page_token);
+        let execute = move |token: String| {
+            let mut builder = self.clone();
+            let req = builder.0.request.clone().set_page_token(token);
+            builder.0.request = req;
```

Comment:
This is considered a partial move:

```
error[E0382]: use of partially moved value: `builder`
    --> src/generated/openapi-validation/src/builders.rs:868:13
     |
867  |             builder.0.request.set_page_token(token);
     |                               --------------------- `builder.0.request` partially moved due to this method call
868  |             builder.send()
     |             ^^^^^^^ value used here after partial move
     |
note: `ListSecretVersionsByProjectAndLocationAndSecretRequest::set_page_token` takes ownership of the receiver `self`, which moves `builder.0.request`
    --> src/generated/openapi-validation/src/model.rs:2611:56
     |
2611 |     pub fn set_page_token<T: Into<Option<String>>>(mut self, v: T) -> Self {
     |                                                        ^^^^
     = note: partial move occurs because `builder.0.request` has type `ListSecretVersionsByProjectAndLocationAndSecretRequest`, which does not implement the `Copy` trait
help: you can `clone` the value and consume it, but this might not be your desired behavior
```

Reply:
but I did simplify the code a little based on your rec